Scarab/Services/ReverseDependencySearch.cs

```diff
@@ -54,6 +54,19 @@ public IEnumerable<ModItem> GetAllEnabledDependents(ModItem item)
         return dependants;
     }
 
+    public bool HasEnabledDependents(ModItem item)
```

Comment:
Instead of making a new function that does the same thing, you could just do GetAllEnabledDependents.Count > 0 or == 0.

Reply:
This function is potentially faster since it doesn't iterate through all items like GetAllEnabledDependents does and it doesn't allocate memory for the list.
Also can be rewritten to be => _items.Values.Any(x => x.EnabledIsChecked && IsDependent(x, item))

Reply:
Is that a meaningful difference tho?

Reply:
Probably not, but I like to optimize things 🤷 . We can also change the GetAllEnabledDependents to return a enumeration instead of a list so that it can be used with Any without an overhead

Reply:
I chose to return list to prevent the bug where enumerable is changed but if that means both can be put in 1 function then idm the change.